.changeset/sharp-pianos-share.md

```diff
@@ -3,7 +3,7 @@
 ---
 
 ---
-updated
+updated:
```

Comment:
Can you add this into the [vscode snippet too](https://github.com/seek-oss/braid-design-system/blob/master/.vscode/Changeset.code-snippets#L7). That's likely the cause